doc/topics/reactor/index.rst

```diff
@@ -27,9 +27,9 @@ event bus is an open system used for sending information notifying Salt and
 other systems about operations.
 
 The event system fires events with a very specific criteria. Every event has a
-:strong:`tag`. Event tags allow for fast top level filtering of events. In
-addition to the tag, each event has a data structure. This data structure is a
-dict, which contains information about the event.
+**tag**. Event tags allow for fast top level filtering of events. In addition
```

Comment:
`top-level`

Reply:
Yeah, this was there before and I did not catch it. Fixed it.